src/adagrams.js

```diff
@@ -1,15 +1,143 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPool = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+  let hand = [];
+  let randomLetter = "";
+  while (hand.length < 10) {
+    randomLetter = getRandomLetter(letterPool);
+    if (letterPool[randomLetter] > 0)  {
+      letterPool[randomLetter] -= 1;
+            hand.push(randomLetter.toUpperCase());
+    }
```

Comment:
Since you are creating letterPool each time this function is created, copying letterPool doesn't matter here, but if in a subsequent refactor you did make letterPool a global constant, you would have to make sure to make a copy!